Themis/Views/Assessment/CorrectionSidebar/FeedbackCellView.swift

```diff
@@ -27,6 +27,11 @@ struct FeedbackCellView: View {
             return Color(.label)
         }
     }
+    
+    var pId: Int?
+    var templatePId: Int?
```

Comment:
I would name these variables `participationId` and `templateParticipationId` so that it's clearer